### DISCUSSION TITLE
How does the encryption in a cloud work

### DISCUSSION THREAD

**Discussion by AirFlowW** [keepassxreboot/keepassxc#9919] [Q&A] [answered]
I wonder how the encryption in a cloud works. Is it okay to have the database in a cloud and open it from there?
Is it like with the master key the file gets completely encrypt? Or does the database gets into the RAM and gets encrypted locally?
I'm pretty interested in that topic. So I'm glad about every information.

**Comment by phoerious** [comment, marked as answer]
The KDBX file is fully encrypted. When you read if from cloud storage, it will only be decrypted locally at your end. So yes, it's fine to put it in a cloud folder as long as you have a strong master password.
